### PR TITLE
Support resources as array and xcassets by @dangthaison91

### LINKED ISSUE #138: Assets not present in the generated project
**Describe the bug**
When using `tuist generate` the resulting project does not include the `Assets.xcassets` folder although in the `Sources` folder.

**To Reproduce**
1. Directory structure
screen shot 2018-09-25 at 11 15 45

2. run `tuist generate`
3. Result
screen shot 2018-09-25 at 11 16 13

**Expected behavior**
`Assets.xcassets`  should be included in the generated project

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: 10.13.6

**Additional context**
Config file:

```
import ProjectDescription

let project = Project(name: "iOSSampleApp",
                      targets: [
                        Target(name: "iOSSampleApp",
                               platform: .iOS,
                               product: .app,
                               bundleId: "sk.kulman.iOSSampleApp",
                               infoPlist: "Info.plist",
                               sources: "Sources/**",
                               dependencies: [
                                    .framework(path: "Carthage/Build/iOS/RxSwift.framework"),
                                    .framework(path: "Carthage/Build/iOS/RxSwiftExt.framework"),
                                    .framework(path: "Carthage/Build/iOS/RxCocoa.framework"),
                                    .framework(path: "Carthage/Build/iOS/RxSwift.framework"),
                                    .framework(path: "Carthage/Build/iOS/FeedKit.framework"),
                                    .framework(path: "Carthage/Build/iOS/Reusable.framework"),
                                    .framework(path: "Carthage/Build/iOS/Defaults.framework"),
                                    .framework(path: "Carthage/Build/iOS/CleanroomLogger.framework"),
                                    .framework(path: "Carthage/Build/iOS/Swinject.framework"),
                                    .framework(path: "Carthage/Build/iOS/SwinjectAutoregistration.framework"),
                                    .framework(path: "Carthage/Build/iOS/SwinjectStoryboard.framework"),
                                    .framework(path: "Carthage/Build/iOS/CRToast.framework"),
                                    .framework(path: "Carthage/Build/iOS/Nuke.framework"),
                                    .framework(path: "Carthage/Build/iOS/RxNuke.framework")
                                ]),
                        Target(name: "iOSSampleAppTests",
                               platform: .iOS,
                               product: .unitTests,
                               bundleId: "sk.kulman.iOSSampleAppTests",
                               infoPlist: "Tests.plist",
                               sources: "Tests/**",
                               dependencies: [
                                    .target(name: "iOSSampleApp"),
                                    .framework(path: "Carthage/Build/iOS/RxTest.framework"),
                                    .framework(path: "Carthage/Build/iOS/RxBlocking.framework"),
                                    .framework(path: "Carthage/Build/iOS/Quick.framework"),
                                    .framework(path: "Carthage/Build/iOS/Nimble.framework")
                               ]),
                        Target(name: "iOSSampleAppUITests",
                               platform: .iOS,
                               product: .unitTests,
                               bundleId: "sk.kulman.iOSSampleUIAppTests",
                               infoPlist: "UITests.plist",
                               sources: "UITests/**",
                               dependencies: [
                                .target(name: "iOSSampleApp"),
                                .framework(path: "Carthage/Build/iOS/SimulatorStatusMagiciOS.framework")
                            ])
                      ])
```

### PR DESCRIPTION
**Pull request by ollieatkinson**
Resolves #138 

### Short description 📝
- Support multi-path sources and resources
- Index xcassets to target

This configuration work perfectly for me:
```swift
sources: ["Sources/**", "Configuration/**"],
resources: ["Tuist-Demo/Storyboard/**/*.{storyboard}", "Tuist-Demo/Resources/**/*.{xcassets,png,strings}"]
```

by @dangthaison91

### CODE REVIEW

**Comment by ollieatkinson** [comment]
Cloned from https://github.com/tuist/tuist/pull/201, work has been rebased and a new fixture test added.

File: features/step_definitions/shared/workspace.rb [hunk 1]
```diff
@@ -7,6 +7,6 @@
   @dir = Dir.mktmpdir
 end
 
-After do |scenario|
+After do |_scenario|
```
Comment (pepibumur) [inline features/step_definitions/shared/workspace.rb]: We can remove `|_scenario|` since we are not using it.

Comment (ollieatkinson) [reply features/step_definitions/shared/workspace.rb]: I don't remember changing this, might have been rubocop. Will remove.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 - Replace swiftlint with swiftformat https://github.com/tuist/tuist/pull/239 by @pepibumur.
 - Bump xcodeproj version to 6.6.0 https://github.com/tuist/tuist/pull/248 by @pepibumur.
 - Fix an issue with Xcode not being able to reload the projects when they are open https://github.com/tuist/tuist/pull/247
+- Support array for `sources` and `resources` paths https://github.com/tuist/tuist/pull/201 by @dangthaison91
```
Comment (dangthaison91) [inline CHANGELOG.md]: How about `by @dangthaison91  and @ollieatkinson` 👍

**Comment by pepibumur** [comment]
All good. Nice job 🎉 

giphy

File: CHANGELOG.md (hunk 2 above)
Comment (ollieatkinson) [reply CHANGELOG.md]: You did all the hard work :) :+1:

**Pull request merged**